packages/contracts-bedrock/src/L1/StandardValidator.sol

```diff
@@ -514,3 +544,203 @@ contract StandardValidatorV180 {
+        if (bytes(_errors).length > 0 && !_allowFailure) {
+            revert(string.concat("StandardValidatorV200: ", _errors));
+        }
+
+        return _errors;
+    }
+
+    function validate(
+        string memory _errors,
+        ISystemConfig _sysCfg,
+        IProxyAdmin _admin,
+        ISuperchainConfig _superchainConfig,
+        IProtocolVersions _protocolVersions,
+        bytes32 _absolutePrestate,
+        uint256 _l2ChainID
+    )
+        internal
+        view
+        returns (string memory)
+    {
+        _errors = assertValidSuperchainConfig(_errors, _superchainConfig, _admin);
+        _errors = assertValidProtocolVersions(_errors, _protocolVersions, _admin);
+        _errors = super.validate(_errors, _sysCfg, _admin, _absolutePrestate, _l2ChainID);
+        return _errors;
+    }
+
+    function assertValidProtocolVersions(
+        string memory _errors,
+        IProtocolVersions _protocolVersions,
+        IProxyAdmin _admin
+    )
+        internal
+        view
+        returns (string memory)
+    {
+        _errors = internalRequire(stringEq(_protocolVersions.version(), protocolVersionsVersion()), "PVER-10", _errors);
+        _errors = internalRequire(
+            _admin.getProxyImplementation(address(_protocolVersions)) == protocolVersionsImpl, "PVER-20", _errors
+        );
+
+        return _errors;
+    }
+
+    function assertValidSuperchainConfig(
+        string memory _errors,
+        ISuperchainConfig _superchainConfig,
+        IProxyAdmin _admin
+    )
+        internal
+        view
+        returns (string memory)
+    {
+        _errors = internalRequire(stringEq(_superchainConfig.version(), superchainConfigVersion()), "SPA-10", _errors);
+        _errors = internalRequire(
+            _admin.getProxyImplementation(address(superchainConfig)) == superchainConfigImpl, "SPA-20", _errors
+        );
+        return _errors;
+    }
+
+    function superchainConfigVersion() public pure returns (string memory) {
+        return "1.2.0";
+    }
+
+    function protocolVersionsVersion() public pure returns (string memory) {
+        return "1.1.0";
+    }
+
+    function systemConfigVersion() public pure override returns (string memory) {
+        return "2.4.0";
+    }
+
+    function l1CrossDomainMessengerVersion() public pure override returns (string memory) {
+        return "2.5.0";
+    }
+
+    function l1StandardBridgeVersion() public pure override returns (string memory) {
+        return "2.2.1";
+    }
+
+    function l1ERC721BridgeVersion() public pure override returns (string memory) {
+        return "2.3.0";
+    }
+
+    function disputeGameFactoryVersion() public pure override returns (string memory) {
+        return "1.0.1";
+    }
+
+    function optimismPortalVersion() public pure override returns (string memory) {
+        return "3.12.0";
+    }
+
+    function optimismMintableERC20FactoryVersion() public pure override returns (string memory) {
+        return "1.10.1";
+    }
```

Comment:
I changed this to `1.10.2` which is wrong since it doesn't match what's currently on develop (i.e. what will become `op-contracts/v2.0.0`), and all tests passed, but I would have expected a test to verify that the semvers in here match what's currently the latest on develop